ros.ws/src/gauge_net/gauge_net/gauge_net/gauge_reader_lite.py

```
@@ -0,0 +1,291 @@
+            self.get_logger().error(
+                f"Error while calling detect: {r.status_code} - {r.text}"
+            )
+            raise Exception(f"Error while calling detect: {r.status_code} - {r.text}")
+
+    def read(self, crop, bbox):
+        self.get_logger().info(f"Calling read on bbox: {bbox}")
+        self.get_logger().info(f"Crop shape: {crop.shape}")
+        _, buf = cv2.imencode(".jpg", crop)
+        b64 = base64.b64encode(buf).decode("utf-8")
+        headers = {"Authorization": f"Bearer {self.token}"}
+        r = requests.post(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)